packages/core/upload/admin/src/utils/appendSearchParamsToUrl.js

```diff
@@ -6,15 +6,15 @@
  * @returns {String} A string representing the URL with the search params appended
  */
 const appendSearchParamsToUrl = ({ url, params }) => {
-  if (url === undefined || params === undefined || typeof params !== 'object' || Object.entries(params).length === 0) {
+  if (url === undefined || typeof params !== 'object') {
     return url;
   }
 
-  const urlObj = new URL(url);
-  const urlParams = urlObj.searchParams;
+  const urlObj = new URL(url, window.strapi.backendURL);
```

Comment:
in our instance (and I guess in many others) `window.strapi.backendURL` is an empty string, so it will still break for us right? This would work: `new URL(url, window.strapi.backendURL || window.location.origin)`

Reply:
Hey! I am on Strapi 4.25.1 and the code it breaks on for me still shows: 
const Rn = new URL(Ye,window.strapi.backendURL); and not the fix you are proposing. If we dont have the change you are proposing, what do I need to set in the .env file or config to get it working?